components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -384,9 +382,9 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
   }
 
   // prevent scrolling on tabs change
-  get height() {
+  get height(): { [cssClass: string]: string; }  {
     return {
-      'min-height': this.scrolling ? 'calc(100vh - 50px)' : 'calc(100vh - 49px)'
+      'min-height': 'calc(100vh - 49px)'
```

Comment:
Since this is now a constant expression there is no reason for it to be in the code-behind; we could just use a normal style in the scss file, right?

Reply:
sure, I am working on this to fix the UX jarring and I will update this PR soon.